apps/full-site-editing/full-site-editing-plugin/editor-gutenboarding-launch/index.ts

```diff
@@ -7,6 +7,7 @@ import { addAction } from '@wordpress/hooks';
 import { dispatch } from '@wordpress/data';
 import { recordTracksEvent } from '@automattic/calypso-analytics';
 import 'a8c-fse-common-data-stores';
+import { isMobile } from '@automattic/viewport';
```

Comment:
I think this would be `<480px` and we'll want to escape already below 768px

https://github.com/Automattic/wp-calypso/blob/master/packages/viewport/src/index.js#L43

Note that there is also `@wordpress/viewport` in case it works better for our need here.

Reply:
> Note that there is also @wordpress/viewport in case it works better for our need here.

Ignoring `@wordpress/viewport` as it seems to be a hook / active listener.

Reply:
Now uses `isWithinBreakpoint( '<768px' )`.

Reply:
Now uses <800px.

Reply:
Now uses `window.innerWidth < 768`. VanillaJS for the win.